docker/Dockerfile

```diff
@@ -1,69 +1,75 @@
-FROM python:3.7.9-alpine3.12
+FROM python:3.8-alpine
```

Comment:
This fails to build for me, as it cannot build the grpcio wheel 🤔 

`ERROR: Failed building wheel for grpcio`

Reply:
This seems to be an issue with building grpcio for M based MacOS, I'm trying to work around it